beta/src/pages/learn/sharing-state-between-components.md

```diff
@@ -73,59 +73,59 @@ h3, p { margin: 5px 0px; }
 
-To coordinate these two panels, you need to "lift their state up" to a parent component in three steps:
+इन दोनों Panels को संचालित करने के लिए, आपको तीन चरणों में इनके "state को लिफ्ट अप" करके इन्हें पैरेंट कौम्पोनॅन्ट में लाना होगा:
 
-1. **Remove** state from the child components.
-2. **Pass** hardcoded data from the common parent.
-3. **Add** state to the common parent and pass it down together with the event handlers.
+1. चाइल्ड कौम्पोनॅन्ट में से state को **हटाना**।
+2. कॉमन पैरेंट में से हार्ड कोड किया हुआ डाटा **पास** करना।
+3. कॉमन पैरेंट में state को फिर से **जोड़ कर** उसे event-handler के साथ पास करना।
 
-This will allow the `Accordion` component to coordinate both `Panel`s and only expand one at a time.
+ऐसा करने से `Accordion` कौम्पोनॅन्ट दोनों `Panel`s को संचालित कर पायेगा और एक समय में उनमें से एक ही expand होगा।
 
-### Step 1: Remove state from the child components {/*step-1-remove-state-from-the-child-components*/}
+### चरण १: चाइल्ड कौम्पोनॅन्ट में से state निकालना {/*step-1-remove-state-from-the-child-components*/}
 
-You will give control of the `Panel`'s `isActive` to its parent component. This means that the parent component will pass `isActive` to `Panel` as a prop instead. Start by **removing this line** from the `Panel` component:
+आप `Panel` के `isActive` का कण्ट्रोल उसके पैरेंट कौम्पोनॅन्ट को देंगे। इसका मतलब है कि पैरेंट कौम्पोनॅन्ट `isActive` को `Panel` तक एक prop की तरह पास करेगा। आप `Panel` कौम्पोनॅन्ट से **यह लाइन हटाने** से शुरुआत कर सकते हैं:
 
 ```js
 const [isActive, setIsActive] = useState(false);
 ```
 
-And instead, add `isActive` to the `Panel`'s list of props:
+और इसके बजाये, `Panel` के props की लिस्ट में `isActive` को जोड़ दें:
 
 ```js
 function Panel({ title, children, isActive }) {
 ```
 
-Now the `Panel`'s parent component can *control* `isActive` by [passing it down as a prop](/learn/passing-props-to-a-component). Conversely, the `Panel` component now has *no control* over the value of `isActive`--it's now up to the parent component!
+अब `Panel` का पैरेंट कौम्पोनॅन्ट `isActive` को [prop की तरह पास कर](/learn/passing-props-to-a-component) के _नियंत्रित_ कर पायेगा। ठीक इसके विपरीत, अब `Panel` कौम्पोनॅन्ट का `isActive` की वैल्यू पर कोई _नियंत्रित नहीं_ रह जायेगा--ये अब पैरेंट कौम्पोनॅन्ट पर निर्भर है!
 
-### Step 2: Pass hardcoded data from the common parent {/*step-2-pass-hardcoded-data-from-the-common-parent*/}
+### चरण २: कॉमन पैरेंट कौम्पोनॅन्ट से हार्ड कोडेड डाटा को पास करना {/*step-2-pass-hardcoded-data-from-the-common-parent*/}
 
-To lift state up, you must locate the closest common parent component of *both* of the child components that you want to coordinate:
+अब state को लिफ्ट अप करने के लिए आपको उस निकटतम कॉमन पैरेंट कौम्पोनॅन्ट का पता लगाना है, जो उन _दोनों_ चाइल्ड कौम्पोनॅन्ट का पैरेंट है जिन्हें आप संचालित करना चाहते हैं:
 
-* `Accordion` *(closest common parent)*
+* `Accordion` _(निकटतम कॉमन पैरेंट)_
```

Comment:
`_(निकटतम कॉमन पैरेंट)_` => `_(निकटतम कॉमन पैरेंट)*`

Reply:
Changed it to *(निकटतम कॉमन पैरेंट)* instead, kindly let me know if its wrong.

Reply:
Yes. That is correct. There was typo with my suggestion. Thank you for pointing that out